challenges/lib/src/main/java/challenges/stacksandqueues/PseudoQueue.java

```diff
@@ -0,0 +1,47 @@
+package challenges.stacksandqueues;
+
+public class PseudoQueue {
+    Stack stackOne;
+    Stack stackTwo;
+
+    public PseudoQueue(){
+        stackOne = new Stack();
+        stackTwo = new Stack();
+    }
+
+    public void push(Stack<Integer> topNode, int newValue) {
```

Comment:
topNode might be a misleading name for your stack.

Reply:
My naming convention needs work sometimes